tests/src/services/users.service.test.ts

```diff
@@ -181,7 +214,7 @@ describe('UsersService tests', () => {
     });
   });
 
-  describe('Storing coupon user by user', () => {
+  describe('storeCouponUsedByUser()', () => {
```

Comment:
This goes directly against what we talked about in the last review. Talk about the use case, not about the function literally